src/module.js

```diff
@@ -59,33 +59,21 @@ var debug = {
     FeatureSelection
 };
 
-// Window can only be set in main thread
-if (Utils.isMainThread) {
-
-    window.Tangram = module.exports = {
-        leafletLayer,
-        debug,
-        version: version.string
-    };
-
-}
-
-if (Utils.isWorkerThread) {
```

Comment:
I'd like access to the `debug` object, but it doesn't need to be under a `Tangram` property per se.